.gitlab-ci.yml

```diff
@@ -50,6 +50,8 @@ RPM:
       - RUNNER:
           - aws/rhel-8.4-x86_64
           - aws/rhel-8.4-aarch64
+          - aws/rhel-8.5-x86_64
+          - aws/rhel-8.5-aarch64
```

Comment:
FTR this is needed by @achilleas-k. I will open another PR where I will replace all mentions of 8.4 with 8.5 and see what happens when we do so.